docs/status.md

```diff
@@ -0,0 +1,58 @@
+# status
+
+```
+cli-utils apply status <dir>
+```
+works for both directories with or without `kustomization.yaml`.
+
+## kustomization directory
+
+First, apply resources from a kustomization directory.
+
+```
+cli-utils apply config/hello
+```
+
+Then, get the status of the applied resources
+```
+cli-utils apply status config/hello
+```
+
+The output is like the following
+```
+2019/08/07 16:38:04 nil value at `volumeClaimTemplates[].metadata.labels` ignored in mutation attempt
```

Comment:
what is this?

Reply:
Ah, this is some warning message when applying the example manifests. I can remove it.